src/components/Followers/Followers.js

```diff
@@ -26,6 +43,15 @@ const Followers = () => {
     })
   }
 
+  const renderFollowing = () => {
+    // console.log(GetFollowingInfo.data.usersFollowers)
+    return GetFollowingInfo.data.userFollowing.map((follower, index) => {
```

Comment:
This is great Bobby, only suggestion is to use consistent indentation on the JSX